cmd/main.go

```diff
@@ -261,6 +264,20 @@ func main() {
 			os.Exit(1)
 		}
 	}
+	if err := (&controller.IutReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
+	}).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "Iut")
+		os.Exit(1)
+	}
+	// nolint:goconst
```

Comment:
Whould be nice to document the reason for this. Also according to https://golangci-lint.run/usage/false-positives/#nolint-directive it should be `//nolint:goconst` without the space after the `//`.